apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -314,6 +317,44 @@ type AutoscalerSpec struct {
 	TargetMemoryUtilization *int32 `json:"targetMemoryUtilization,omitempty"`
 }
 
+// Probe defines the OpenTelemetry's pod probe config. Only Liveness probe is supported currently.
+type Probe struct {
```

Comment:
Can we enforce these minimum values via the validator?

Reply:
validator added.

Reply:
instead of using the struct tag can you add the validations to https://github.com/open-telemetry/opentelemetry-operator/blob/6825b81441070908a467dc8704d0f3db1af94c6d/apis/v1alpha1/opentelemetrycollector_webhook.go#L119

Reply:
Done.